ModelarDB-Evaluator/src/main.rs

```diff
@@ -206,7 +206,7 @@ fn download_update_and_compile_modelardb() -> Result<(), IOError> {
 
 /// Read the Apache Parquet file at `uncompressed_data_path`, remove columns that contain NULL
 /// values, and ensure that the types used for its columns are types supported by modelardbd.
-/// Returns [`ParquetError`] if the file cannot be read or the data in it normalized.
+/// Returns [`ParquetError`] if the file cannot be read or the data in it cannot normalized.
```

Comment:
Very minor but missing `be` in `cannot normalized`.

Reply:
Fixed.